tests/system/providers/pinecone/example_dag_pinecone.py

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import os
+from datetime import datetime
+
+from airflow import DAG
+from airflow.providers.pinecone.operators.pinecone_operator import PineconeIngestOperator
+
+index_name = os.getenv("INDEX_NAME", "test")
+namespace = os.getenv("NAMESPACE", "example-pinecone-namespace")
+
+
+with DAG(
+    "example_pinecone_ingest",
+    schedule_interval=None,
+    start_date=datetime(2023, 1, 1),
+    catchup=False,
+) as dag:
```

Comment:
```suggestion
with DAG(
    "example_pinecone_ingest",
    schedule=None,
    start_date=datetime(2023, 1, 1),
    catchup=False,
):
```
A little "modernization" of syntax and params.